core/deployment/banned-signatures.txt

```diff
@@ -1,5 +1,8 @@
 @defaultMessage Don't use Maven classes. They won't be available when using Gradle.
 org.apache.maven.**
 org.codehaus.plexus.**
+
 @defaultMessage Never use Type#toString() as it's almost always the wrong thing to do. Usually org.jboss.jandex.DotName#toString() is what is needed
 org.jboss.jandex.Type#toString()
+
+org.apache.commons.io.** @ Don't use commons-io dependency
```

Comment:
I'm probably missing something, but why aren't we using maven-enforcer's `bannedDependencies`?

Reply:
Indeed. I'm creating a PR.